ui/src/app/utils/navigation.tsx

```diff
@@ -91,24 +108,14 @@ export const useNavigation = () => {
     });
   };
 
-  const navigate = (commands, extras?: NavigateExtras) => {
-    navigateByUrl(commands.join('/'), extras);
+  const navigate = (path: string | string[], extras?: NavigateExtras) => {
```

Comment:
Do we have any usages of navigate with a single string?

Reply:
I thought we did but it appears no!  I updated this to require an array instead.